project/views.py

```diff
@@ -30,23 +30,31 @@ def register_user():
     last_name = request_data.get('last_name')
     other_names = request_data.get('other_names')
     email = request_data.get('email')
-    tel = request_data.get('tel')
+    telephone = request_data.get('telephone')
     user_name = request_data.get('user_name')
     password = request_data.get('password')
+    valid_user = incident_validator.validate_user_credentials(
+        email, password, user_name, telephone)
+    if valid_user:
```

Comment:
@3Nakajugo With this line, if the user is valid, why return that some fields are missing.